.idea/codeStyleSettings.xml

```diff
@@ -291,6 +291,34 @@
         <option name="DOWHILE_BRACE_FORCE" value="3" />
         <option name="WHILE_BRACE_FORCE" value="3" />
         <option name="FOR_BRACE_FORCE" value="3" />
+        <Objective-C-extensions>
```

Comment:
The IDE adds this automatically, if you rebase this will go away (I just committed this I think).